files/pnp/passwd.txt

```diff
@@ -0,0 +1 @@
+nagiosadmin:$apr1$zM9FEamr$rFQlBm8uenm.oFA9mi8o90
```

Comment:
@walterheck I've already fix this, username and password is password is passed to class. I will remove this file 'files/pnp/passwd.txt' in the next commit